### PR TITLE
found baby bug, heres the fix

### PR DESCRIPTION
**Pull request by ryanaolea**

### CODE REVIEW

File: client/src/App.jsx [hunk 1]
```diff
@@ -37,12 +37,12 @@ const findId = (element) => {
...
   return id;
 };
-
+// 40344
 class App extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      product_Id: 40344,
+      product_Id: 40359,
```
Comment (FlightfulKiwi) [inline client/src/App.jsx]: 40344 is 'Camo Onesie'; 40359 is 'Marcelina Cap'. Why the change here? If the product name is hard-coded, it would need to match the ID, yeah?

**Review by FlightfulKiwi — COMMENTED** [review]
See one comment. Gimme ur thoughts and we good.

File: client/src/App.jsx (hunk 1 above)
Comment (ryanaolea) [reply client/src/App.jsx]: i was testing for data purposes. the code i just pushed up is not this. fyi

**Review by FlightfulKiwi — APPROVED** [review]
